v2/pages/adminAssessment/adminAssessment.sql

```diff
@@ -23,6 +23,44 @@ WHERE
     AND q.deleted_at IS NULL
 ORDER BY z.number, z.id, aq.number;
 
+-- BLOCK question_stats
+SELECT
+    sub.question_id,
+    avg(sub.mean_score_per_user_per_question) as mean_score_per_question,
+    sub.number,
```

Comment:
In the current `master` we are numbering questions by `<%= question.alternative_group_number %>`, if there is only one alternative, or by `<%= question.alternative_group_number %>.<%= question.number_in_alternative_group %>` if there are multiple alternatives. See [adminAssessment.ejs:69](https://github.com/PrairieLearn/PrairieLearn/blob/master/v2/pages/adminAssessment/adminAssessment.ejs#L69) for an example. In that code we are doing this in the EJS file, but it might be cleaner to compute the question number in the SQL. We might even want to add a function to `sprocs/` that is something like `admin_assessment_question_number(assessment_question_id)`. For an example of this new numbering scheme in action, see TAM 212, Fall 2016, Quiz 3 in the updated DB dump that I will be sending you shortly.